playbooks/osp/12/overcloud/overcloud-deploy.sh.j2

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
need to `set -o pipefail` otherwise if `openstack overcloud deploy ...` fails but tee succeeds, then the associated `Overcloud deploy` Ansible task will "succeed", though really failing.

i suggest also `set -e` so that if any command fails then the script immediately exits with a failure (eg `source` fails because `/home/stack/stackrc` doesn't exist for whatever reason).

Reply:
`set -v` and/or `set -x` can also be helpful if a shell script is templated (as this one is) to know what the final executed script looked like (as sometimes the template logic and variables can be hard to ascertain due to intermediate/computed variables, though not necessarily applicable in this specific case).